### PR TITLE
Use `String.prototype.replaceAll()` where appropriate

### PR DESCRIPTION
**Pull request by Snuffleupagus**
This fairly new method allows replacing *multiple* occurrences within a string without having to use regular expressions.

Please refer to:
 - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replaceAll
 - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replaceAll#browser_compatibility

### CODE REVIEW

File: external/importL10n/locales.js [hunk 1]
```diff
@@ -27,7 +27,7 @@ const DEFAULT_LOCALE = "en-US";
 const EXCLUDE_LANG_CODES = ["ca-valencia", "ja-JP-mac"];
 
 function normalizeText(s) {
-  return s.replace(/\r\n?/g, "\n").replace(/\uFEFF/g, "");
+  return s.replace(/\r\n?/g, "\n").replaceAll("\uFEFF", "");
```
Comment (calixteman) [inline external/importL10n/locales.js:30]: Why don't you replace the first `replace` by a `replaceAll` ?
I guess it doesn't matter that much but at least it's more explicit than a `g` regex.

**Review by calixteman — APPROVED** [review]
LGTM.

File: external/importL10n/locales.js (hunk 1 above)
Comment (Snuffleupagus) [reply external/importL10n/locales.js:30]: Do you mean simply changing `replace` into `replaceAll`, or something else?

Note that I don't *think* that we can easily replace the regular expression with a string, considering what it does. Unless I'm missing something the replacement currently does the following, which can't be captured with just a string:
 - `\r` -> `\n`
 - `\r\n` -> `\n`

Comment (calixteman) [reply external/importL10n/locales.js:30]: `replaceAll` can take a regex but I missed in the docs that it has to have a `g`...
But as said, it'd slightly improve the readability of the code, that said it's an opinion, so do whatever you want with it.

Comment (calixteman) [reply external/importL10n/locales.js]: Yep it's fine, r=me.
And `replaceAll` throws when the `g` is missing so it could help to avoid to overlook it.

**Comment by Snuffleupagus** [comment]
*Hopefully the bots use a modern enough Node.js version for this to work...*

/botio preview

**Pull request merged**